docs/chapters/04_features.md

```diff
@@ -487,6 +487,52 @@ mutation {
 
 > [!NOTE]  Remember that, in case you want to subscribe to a read model that is restricted to a specific set of roles, you must send the **access token** retrieved upon sign-in. Check ["Authorizing operations"](#authorizing-operations) to know how to do this.
 
+
+### Validating read models' filters
+You can validate read models by using the `before` hook. The functions you add here will be executed **before fetching from your event-store**:
+
+```typescript
+@ReadModel({
+  authorize: [User],
+  before: [CartReadModel.validateUser],
+})
+export class CartReadModel {
+  public constructor(
+    readonly id: UUID,
+    readonly userId: UUID
+  ) {}
+
+  public static validateUser(filter: FilterFor<CartReadModel>, currentUser?: UserEnvelope): FilterFor<CartReadModel> {
+    if (filter?.userId !== currentUser.id) throw NotAuthorizedError("...")
```

Comment:
Remember that filters are no longer optional, so you can remove the `?` after filter